services/vision/mlvision/ml_model.go

```diff
@@ -73,33 +73,37 @@ func registerMLModelVisionService(
 
 	classifierFunc, err := attemptToBuildClassifier(mlm)
 	if err != nil {
-		logger.Infow("error turning ml model into a classifier", "model", params.ModelName, "error", err)
+		logger.Debugw("unable to use ml model as a classifier, will attempt to evaluate as"+
+			"detector and segmenter", "model", params.ModelName, "error", err)
```

Comment:
specify that it is a 3D segmenter, as when most people hear "segmenter" they would assume 2D segmentation. Same for the Debug logs below